spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -28,7 +28,8 @@
     let!(:leaderships) { create_list(:leadership, 3) }
     let(:vacancy) do
       VacancyPresenter.new(build(:vacancy, :complete,
-                                 pay_scale: pay_scales.sample,
+                                 min_pay_scale: pay_scales.sample,
```

Comment:
Has this change been driven through tests as with this as the only change to the integration spec it alone does not produce a failing spec which I believe has led to a feature that looks tested but doesn't work when the user tries to do it.

I think we can improve this test file by removing the vacancy object as I assume it is the part that is to blame for the false confidence. Feature tests that are concerned with testing the full flow of creating a particular record in the database should probably not build objects of the same. Perhaps we should pass a hash of explicit parameters to our form helpers that help us fill in the pages and test for them exactly at the end of the process?

When I add values, they don't appear:
![screen shot 2018-05-02 at 12 11 16](https://user-images.githubusercontent.com/912473/39521241-a3d3a7d0-4e05-11e8-804c-aa9564595ea9.png)

When I edit the vacancy they are blank:
![screen shot 2018-05-02 at 12 12 01](https://user-images.githubusercontent.com/912473/39521248-a8f707d4-4e05-11e8-94d7-360708cc9d5a.png)